WeakAuras/Transmission.lua

```diff
@@ -1642,11 +1642,18 @@ function WeakAuras.ShowDisplayTooltip(data, children, matchInfo, icon, icons, im
   if not IsAddOnLoaded('WeakAurasOptions') then
     LoadAddOn('WeakAurasOptions')
   end
-
-  local ok,thumbnail = pcall(regionOptions[regionType].createThumbnail, thumbnailAnchor, regionTypes[regionType].create);
+  if thumbnailAnchor.currentThumbnail then
+    thumbnailAnchor.currentThumbnail:Hide()
+    thumbnailAnchor.currentThumbnail = nil
+  end
+  local ok,thumbnail = pcall(regionOptions[regionType].createThumbnail, thumbnailAnchor);
   if not ok then
     error("Error creating thumbnail", 2)
   end
+  ok = pcall(regionOptions[regionType].modifyThumbnail, thumbnailAnchor, thumbnail, data)
+  if not ok then
+    error("Error modifying thumbnail", 2)
```

Comment:
This isn't actually all that useful. It would probably be helpful to pass along the error message, so that if we ever break the thumbnail functions, it isn't a nightmare to debug.

Or, alternatively since this halts execution, don't use pcall at all.

Reply:
Should i remove pcall for createThumbnail too ?

Reply:
Thinking more about it, because of Classic, and high probability that sometime someone import an aura using an icon that is not in game files, I'd prefer if it fails silently and doesn't halt the display of import tooltip